components/progressbar/stories/meter.stories.js

```diff
@@ -1,37 +1,91 @@
  */
 export default {
 	title: "Meter",
 	component: "ProgressBar",
 	argTypes: {
 		...ProgressBar.argTypes,
+		size: size(["s", "l"]),
+		isIndeterminate: { table: { disable: true } },
 		fill: {
 			name: "Fill color",
 			type: { name: "string" },
+			defaultValue: "info",
 			table: {
 				type: { summary: "string" },
 				category: "Component",
+				defaultValue: { summary: "info" },
 			},
-			options: ["notice", "positive", "negative"],
+			options: ["info", "notice", "positive", "negative"],
 			control: "select",
 		},
 	},
-	args: ProgressBar.args,
+	args: {
+		...ProgressBar.args,
+		size: "l",
+		label: "Storage space",
+	},
 	parameters: {
 		packageJson: pkgJson,
 	},
 };
 
+/**
+ * By default, meters have a blue fill and are the large sizes.
+ * 
+ * Meters should always have a label. In rare cases where context is sufficient and an accessibility expert has reviewed the design, the label could be undefined. These meters without a visible label should still include an “aria-label” or “aria-labelledby” in HTML, depending on the context. The label is always placed above the track.
+ * 
+ * When the label is too long for the available horizontal space, it wraps to form another line. The value is always shown in full and never wraps or truncates.
+ */
 export const Default = MeterGroup.bind({});
 Default.args = {
 	value: 50,
-	label: "Storage space",
+};
+
+/**
+ * Meters come in two sizes: large and small. By default, meters are large. Use the small size when there are multiple meters shown at the same time in a more confined space, such as in tables or cards.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withBorder: false,
+	withHeading: false,
+	...args,
+}, context);
+Sizing.args = Default.args;
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * Meter variants can be used to represent semantic values by switching variants as the value changes, from positive, to notice, and then to negative. This kind of variant switching should be handled appropriately within the context of your product so that you’re setting accurate expectations for your users about the semantic meaning.
+ * 
+ * By default, the meter has a informative, blue fill to show the value. This can be used to represent a neutral or non-semantic value. The positive variant has a green fill, representing a positive semantic value. The notice variant has an orange fill, and can be used to warn users about a situation that may need to be addressed soon. The negative variant has a red fill, and can be used to warn users about a critical situation that needs their urgent attention.
+*/
+export const FillColors = (args, context) => FillGroup({
+	variants: ["info", "positive", "negative", "notice"],
```

Comment:
"Positive" seems correct to me and matches the guidelines too, but any idea why the green variant is called "Neutral" on the Docs site?

<img width="360" alt="image" src="https://github.com/user-attachments/assets/80d197aa-da26-4734-88d9-0b84c54ab302">

Reply:
Nope, I think it's fine as it is.